libraries/GCS_MAVLink/GCS.h

```diff
@@ -129,6 +129,13 @@ class GCS_MAVLINK
     // call to reset the timeout window for entering the cli
     void reset_cli_timeout();
 
+    bool is_usb() { return chan == MAVLINK_COMM_0; }
```

Comment:
Of course, then we end up with this:

```
    if (link.is_high_bandwidth() && hal.gpio->usb_connected()) {
```

Which is even  worse than what was there :-)

Oh well.